packages/dcos-integration-test/extra/api_session_fixture.py

```diff
@@ -2,13 +2,22 @@
 DcosApiSession object that will be injected into the pytest 'dcos_api_session' fixture
 via the make_session_fixture() method
 """
+from test_helpers import expanded_config
+
 from test_util.dcos_api_session import DcosApiSession, DcosUser, get_args_from_env
 from test_util.helpers import CI_CREDENTIALS
 
 
 def make_session_fixture():
     args = get_args_from_env()
-    args['auth_user'] = DcosUser(CI_CREDENTIALS)
-    dcos_api_session = DcosApiSession(**args)
+
+    exhibitor_admin_password = None
+    if expanded_config['exhibitor_admin_password_enabled'] == 'true':
+        exhibitor_admin_password = expanded_config['exhibitor_admin_password']
+
+    dcos_api_session = DcosApiSession(
+        auth_user=DcosUser(CI_CREDENTIALS),
```

Comment:
Our tooling apparently violates the style guide. Here's what the code quality check tells me when I double indent this line:

```
(dcos) branden@crateria dcos$ tox -e py35-syntax
<snip>
./packages/dcos-integration-test/extra/api_session_fixture.py:19:13: E126 continuation line over-indented for hanging indent
flake8.main.application   MainProcess   3377 INFO     Found a total of 28 violations and reported 1
ERROR: InvocationError: '/Users/branden/Code/dcos/.tox/py35-syntax/bin/flake8 --verbose'
______________________________________________________________ summary _______________________________________________________________
ERROR:   py35-syntax: commands failed
(dcos) branden@crateria dcos$
```

Our tooling needs to be updated to reflect the style guide. Also CI should be telling me about style violations, not comments.